$NetBSD: patch-ah,v 1.6 2019/01/31 14:15:55 wiz Exp $

date: 2005-10-13 21:15:58 +0900;  author: markd;  state: Exp;
The latest doxygen encodes the source directory path into some of the
output filenames.  This is problematic in a pkgsrc context as it means
WRKDIR pathnames can appear in generated filenames.  Fix by reverting
to the old way of naming the files.  Bump PKGREVISION.

--- src/dirdef.cpp.orig	2012-05-28 23:17:06.000000000 +0900
+++ src/dirdef.cpp	2012-06-23 11:52:57.000000000 +0900
@@ -108,7 +108,8 @@
 {
   //printf("DirDef::getOutputFileBase() %s->dir_%s\n",
   //    m_diskName.data(),encodeDirName(m_diskName).data());
-  return "dir_"+encodeDirName(m_diskName);
+  //return "dir_"+encodeDirName(name());
+  return QCString().sprintf("dir_%06d",m_dirCount);
 }
 
 void DirDef::writeDetailedDescription(OutputList &ol,const QCString &title)
